README.md

```diff
@@ -148,6 +148,7 @@ The resolve handler is documented in the [shortstop-resolve](https://github.com/
 
 
 #### Environment-aware
+Defaults to `config.json` and overrides come from environment specific `.json` files based on `NODE_ENV`.
```

Comment:
This doesn't say anything different from the paragraph below. Though I would say that perhaps this section would be more understandable if written as a bulleted list describing the reading of config.json and any override files.